### PR TITLE
changed to idle screen while loading, removed startscren buttons

### PR DESCRIPTION
**Pull request by pIlIp-d**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -5,6 +5,8 @@ import ShipmentGame from './components/minigame/shipment/ShipmentGame';
 import { LoadingScreen } from './components/startscreen/LoadingScreen';
```
Comment (timowenz) [inline src/App.tsx:5]: Remove unused import

File: src/components/startscreen/LoadingScreen.tsx [hunk 2]
```diff
@@ -8,21 +8,7 @@ type LoadingScreenProps = {
...
-	}
-	function showOverworld() {
-		if (progress === 100) {
-			setScene(Scene.Overworld);
-		}
-	}
+
 	return (
 		<>
 			<div onClick={showOverworld} onTouchStart={showOverworld} className="loadingScreen">
```
Comment (timowenz) [inline src/components/startscreen/LoadingScreen.tsx:14]: Where is function showOverworld?

File: src/components/ui/ImageScreen.tsx [hunk 3]
```diff
@@ -6,11 +6,12 @@ type ImageScreenProps = {
 	onclick?: () => void;
 	backButton?: boolean;
 	init?: () => void;
+	opacity?: number;
 };
-export default function ImageScreen({ imageSource, onclick = () => {}, backButton = false, init }: ImageScreenProps) {
+export default function ImageScreen({ imageSource, onclick = () => { }, backButton = false, init, opacity = 1 }: ImageScreenProps) {
```
Comment (timowenz) [inline src/components/ui/ImageScreen.tsx:11]: An empty arrow function? 
'onclick = () => {}'

**Review by timowenz — CHANGES_REQUESTED** [review]
Run Lint

File: src/components/ui/ImageScreen.tsx (hunk 3 above)
Comment (pIlIp-d) [reply src/components/ui/ImageScreen.tsx:11]: dann müssen wir nicht conditional insert im tsx part machen

File: src/components/startscreen/LoadingScreen.tsx (hunk 2 above)
Comment (pIlIp-d) [reply src/components/startscreen/LoadingScreen.tsx:14]: logic made it simpler -> not needed as function

File: src/App.tsx (hunk 1 above)
Comment (pIlIp-d) [reply src/App.tsx:5]: lint is coming

**Review by JulianWer — APPROVED** [review]

**Pull request merged**